packages/component-library/draft/Kaizen/Modal/Presets/ConfirmationModal.scss

```diff
@@ -19,6 +20,8 @@
   padding: $ca-grid $ca-grid $ca-grid * 1.5;
   text-align: center;
   background-repeat: no-repeat;
+  border-radius: $kz-border-solid-border-radius $kz-border-solid-border-radius 0
+    0;
```

Comment:
![image](https://user-images.githubusercontent.com/702885/78869162-775d5380-7a87-11ea-81a5-7d39603b4b78.png)

It's a bit glitchy becuase of the double border. `overflow-hidden` can be used, but again, it's risky that it might clip tooltips, etc.

I wonder if there's a better way to solve the double border border problem? One thing I tried is making the outer border radius much bigger (eg `15px` so that it's hidden). This looks better, but it's a bit of a hack and not maintainable. Maybe there's a good CSS trick someone's figured out. Maybe it's better to leave this square like before for now (the second image below) - could revisit when we do a full uplift and refactor of modals.

Reply:
Noting that bleed is acceptable in this scenario: https://trello.com/c/8UJbdJ6z/875-uplift-border-radius-in-kaizen